cmds/import_image/import_image_cmd.go

```diff
@@ -4,20 +4,47 @@ import (
 	"errors"
 	"fmt"
 
-	common "github.com/cloudfoundry-community/bosh-softlayer-tools/common"
+	"github.com/cloudfoundry-community/bosh-softlayer-tools/common"
 
+	boshlog "github.com/cloudfoundry/bosh-utils/logger"
+	boshretry "github.com/cloudfoundry/bosh-utils/retrystrategy"
 	sldatatypes "github.com/maximilien/softlayer-go/data_types"
-	softlayer "github.com/maximilien/softlayer-go/softlayer"
+	"github.com/maximilien/softlayer-go/softlayer"
+	"github.com/pivotal-golang/clock"
+	"time"
+)
+
+var (
+	locations = []sldatatypes.SoftLayer_Location{
```

Comment:
Why are the locations fixed in code. Add them to a JSON file that can be loaded or passing them in some flag.

If these are defaults then they need to be Golang constants.